bcda/models/models.go

```diff
@@ -96,7 +98,7 @@ type User struct {
 	Name  string    `json:"name"`                                   // name
 	Email string    `json:"email"`                                  // email
 	Aco   ACO       `gorm:"foreignkey:AcoID;association_foreignkey:UUID"`
-	AcoID uuid.UUID `gorm:"type:char(36)" json:"aco_id"` // aco_id
+	AcoID uuid.UUID `gorm:"type:char(36)" json:"aco_id"` 			// aco_id
```

Comment:
Can we get the ID from the ACO model instead of having an additional field?

Reply:
I think you're asking about the semantics of GORM. How to describe associations with GORM: http://doc.gorm.io/associations.html

We are using the 'belongs to' approach. We could possibly have used the 'has one' approach described later in the page, but I don't know whether or how the resulting tables and constraints would differ.